orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandlerTest.kt

```diff
@@ -1839,6 +1842,74 @@ object RunTaskHandlerTest : SubjectSpek<RunTaskHandler>({
       }
     }
   }
+
+  describe("when a previous stage was a skipped manual judgment stage with auth propagated") {
+    given("a stage with a manual judgment type and auth propagated") {
+      val timeout = Duration.ofMinutes(5)
+      val lastModifiedUser = StageExecution.LastModifiedDetails()
+      lastModifiedUser.user = "user"
+      lastModifiedUser.allowedAccounts = listOf("user")
+
+      val pipeline = pipeline {
+        stage {
+          refId = "1"
+          type = manualJudgmentStage.type
+          manualJudgmentStage.plan(this)
+          status = SUCCEEDED
+          context["manualSkip"] = true
```

Comment:
The tests still work without this line too.

Reply:
Removed